building.md

```diff
@@ -0,0 +1,34 @@
+# Building NV2
```

Comment:
Do you want to rename nv2 to notation in this pr?

Reply:
Good catch as I forgot to update this file.